scripts/test/e2e/run

```diff
@@ -15,7 +15,11 @@ function fetch_images {
 
 function setup {
     local project=$1
-    COMPOSE_PROJECT_NAME=$1 COMPOSE_FILE=$2 docker-compose up --build -d >&2
+    local file=./e2e/compose-env.yaml
+
+    DOCKERD_EXPERIMENTAL="${DOCKERD_EXPERIMENTAL:-}"
+    test "${DOCKERD_EXPERIMENTAL}" -eq "1" && file="${file}:./e2e/compose-env.experimental.yaml"
```

Comment:
Unless you wanted to subsequently `export` it you could inline the `${DOCKERD_EXPERIMENTAL:-}` into the usage in the `test`, but that's a very minor nit.